src/common/sorted-table/index.js

```diff
@@ -569,21 +594,13 @@ export default class SortedTable extends Component {
     const { target } = event
     this._selectItem(+target.name, target.checked, event.nativeEvent.shiftKey)
   }
-
-  _executeGroupedAction = handler => {
-    const { state } = this
-    return handler(
-      state.all
-        ? this._getItems()
-        : filter(this._getItems(), item => state.selectedItemsIds.has(item.id))
-    )
-  }
-
-  _executeRowAction = event => {
-    const { props } = this
-    const item = this._getVisibleItems()[event.currentTarget.dataset.index]
-    props.rowAction(item, props.userData)
-  }
+  _getSelectedItems = createSelector(
+    () => this.state.all,
+    () => this.state.selectedItemsIds,
+    () => this._getItems(),
```

Comment:
Why not `this._getItems` directly?

Reply:
We have an error undefined for `_getItems`  because it' s defined in constructor.

Reply:
Yes, you may need to move this selector in the constructor as well.